Prompting local LLMs throws exception

Version: 0.2.13
Using local LLMs is broken. When the response is returned a nullpointexception occurs:
```
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "dev.langchain4j.model.output.TokenUsage.inputTokenCount()" because "tokenUsage" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:722)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.NullPointerException: Cannot invoke "dev.langchain4j.model.output.TokenUsage.inputTokenCount()" because "tokenUsage" is null
	at com.devoxx.genie.model.request.ChatMessageContext.setTokenUsageAndCost(ChatMessageContext.java:49)
	at com.devoxx.genie.service.NonStreamingPromptExecutor.lambda$execute$0(NonStreamingPromptExecutor.java:44)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	... 5 more
```

Fix is on its way...

--- a/src/main/java/com/devoxx/genie/model/request/ChatMessageContext.java
+++ b/src/main/java/com/devoxx/genie/model/request/ChatMessageContext.java
@@ -44,8 +44,11 @@ public class ChatMessageContext {
         return totalFileCount > 0;
     }
 
-    public void setTokenUsageAndCost(TokenUsage tokenUsage, double inputCost, double outputCost) {
+    public void setTokenUsageAndCost(TokenUsage tokenUsage) {
         this.tokenUsage = tokenUsage;
-        this.cost = (tokenUsage.inputTokenCount() * inputCost + tokenUsage.outputTokenCount() * outputCost) / 1_000_000.0;
+        if (this.tokenUsage != null) {
+            this.cost = (tokenUsage.inputTokenCount() * languageModel.getInputCost() +
+                    tokenUsage.outputTokenCount() * languageModel.getOutputCost()) / 1_000_000.0;
+        }
     }
 }

--- a/src/main/java/com/devoxx/genie/service/NonStreamingPromptExecutor.java
+++ b/src/main/java/com/devoxx/genie/service/NonStreamingPromptExecutor.java
@@ -1,7 +1,6 @@
 package com.devoxx.genie.service;
 
 import com.devoxx.genie.error.ErrorHandler;
-import com.devoxx.genie.model.LanguageModel;
 import com.devoxx.genie.model.request.ChatMessageContext;
 import com.devoxx.genie.ui.panel.PromptOutputPanel;
 import com.intellij.openapi.diagnostic.Logger;
@@ -40,12 +39,7 @@ public class NonStreamingPromptExecutor {
                     chatMessageContext.setAiMessage(response.content());
 
                     // Set token usage and cost
-                    LanguageModel languageModel = chatMessageContext.getLanguageModel();
-                    chatMessageContext.setTokenUsageAndCost(
-                        response.tokenUsage(),
-                        languageModel.getInputCost(),
-                        languageModel.getOutputCost()
-                    );
+                    chatMessageContext.setTokenUsageAndCost(response.tokenUsage());
 
                     promptOutputPanel.addChatResponse(chatMessageContext);
                 } else if (isCancelled) {

